meetings/2022-11-30.md

```diff
@@ -0,0 +1,156 @@
+
+### Trust between owners on same origin with multiple storages
+URL: https://github.com/solid/specification/pull/290
+
+* SC: This is a clarification that we're trying to get from one of earlier PRs around storage, owners, and origin. Following up on [issue #267](https://github.com/solid/specification/issues/267). Brought it into the meeting to create awareness and more engagement because it's a change that is expected to be part of the next release. It is a consideration we're trying to take up, whether or not it makes it into the release. 
+* SC: The idea is that if a server offers multiple storages, should the owners of each storage have trust between them? There were no objections, one approval, but there were comments which requested clarity on the intention. I think we need better justification/language to clarify those concerns, or we don't include in next release. 
+* SC: This was worked on with TBL, KK, JB, and myself. Maybe others can offer more insight. 
+* TBL: Very obscure point, important but "trust between..." is ambiguous. Does it mean you should not build systems where there is no trust? Does it mean that there will be trust between ??? necessarily? The way it is stated is very confusing, not helpful.
+* SC: Good self-review because we drafted together :) 
+* TBL: Yes, and I thought that when we did.
+* SC: First part of this is a note. Second part is ??? That is the one where commenter felt it was unclear. If we focus on that we can move this forward. Line 904.
+* TBL: ??? about web app security? If people have have webpages with JS on their pods, the issue ??? Solid doesn't use web origins, but when people use solid pods are ordinary blogs, for example.. 
+* SC: Pre-proposal: would you (TBL) or anyone object to not including that statement at all?
+* TBL: Happy to leave the whole PR out of next release.
+* SC: No objections to the note and people find it useful. It is the second part. I can update PR to remove that. Would you approve?
+* TBL: Yes. 
+
+ACTION: SC to update PR to reflect the discussion (remove second part, leave note).
+
+
+### Agent-specific Discovery
+URL: https://woutermont.github.io/solid/agent-specific-discovery.html
+
+* WT: Proposal is based on stuff already in SAI, but tries to take out a specific part that doesn't belong with the rest, and make it a separate specification. It is concerned with agents/client who want to discover info about some entity; they can not only find public info available in identity document, but can also find an easy way to know where the info specific to that client lives. 
+* WT: One approach would be to have a lot of `seeAlso`s in the profile document and have the client check all of them. ??? proposes to have a microservice/hub to which an entity within their profile doc points, and the client can ask the service for a pointer to the place where their specific data can be found. 
```

Comment:
```suggestion
* WT: One approach would be to have a lot of `seeAlso`s in the profile document and have the client check all of them. Because this is not performantly scalable, this document proposes to have a microservice/hub to which an entity points within their profile doc, and the client can ask the service for a pointer to the place where their specific data can be found. 
```